easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'PythonBundle'
+
+name = 'rasterio'
+version = '1.2.3'
+
+homepage = 'https://github.com/mapbox/rasterio'
+description = "Rasterio reads and writes geospatial raster data."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),  # for numpy
+    ('matplotlib', '3.3.3'),
+    ('GDAL', '3.2.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('affine', '2.3.0', {
+        'checksums': ['2e045def1aa29e613c42e801a7e10e0b9bacfed1a7c6af0cadf8843530a15102'],
+    }),
+    ('attrs', '20.3.0', {
+        'modulename': 'attr',
+        'checksums': ['832aa3cde19744e49938b91fea06d69ecb9e649c93ba974535d08ad92164f700'],
+    }),
```

Comment:
Python 3.8.6 provides `attrs` 20.2.0 (https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.8.6-GCCcore-10.2.0.eb#L226) - do we need the slightly newer version here?

Reply:
indeed, it's not necessary